NAMESPACE

```diff
@@ -1 +1,7 @@
-exportPattern("^[[:alpha:]]+")
+# Generated by roxygen2: do not edit by hand
```

Comment:
Hi Andrea, I generated them with the devtools package. Instead of us writing the NAMESPACE, devtools does it for us, using any packages that we have in Roxygen code ("#' @importFrom dplyr mutate " etc.). 
To be honest, it's not that important for this package, because we only have one function and we are not importing a lot from other packages, so if you want to take out the Roxygen code, feel free. I think it's useful for creating the documentation though :-) 

p.s. sorry about the author thing! I had forked the repo before you did that :-)